CSharpMath/Structures/Dictionary.cs

```diff
@@ -132,11 +137,9 @@ public class LaTeXCommandDictionary<TValue> : ProxyAdder<string, TValue>, IEnume
   /// allowing fast lookup of the first TFirst corresponding to any TSecond,
   /// in addition to the usual lookup of a TSeconds by a TFirst.
   /// </summary>
-#pragma warning disable CA1710 // Identifiers should have correct suffix
-#pragma warning disable CA1010 // Collections should implement generic interface
+  [SuppressMessage("Naming", "CA1710:Identifiers should have correct suffix", Justification = NotACollection)]
+  [SuppressMessage("Design", "CA1010:Collections should implement generic interface", Justification = NotACollection)]
```

Comment:
The `ProxyAdder` justification of "not actually a collection" is different from "It's two dictionaries in one so a single IReadOnlyDictionary interface isn't appropriate. Instead both are provided.".

Reply:
For me, without the suppression it complained about not having a Collection rather than having Dictionary in the name. But can I leave this to you?

Reply:
CA1710 suggests Collection suffix for IEnumerable and ICollection, and suggests Dictionary suffix for IDictionary. Since we don't implement IDictionary anymore, of course it will warn.